string url
float32 prvaStevilka
float32 drugaStevilka
---
string cepivo
